app/models/concerns/paperclip_ar.rb

```diff
@@ -0,0 +1,31 @@
+# paperclip needs a bunch of AR defined to work
+module PaperclipAr
```

Comment:
I'm wondering if this should just live in mixins to be consistent and not introduce a new directory.  I know "concerns" is the new default, but I'm thinking let's be deliberate when we move things there.